src/Tasks.js

```diff
@@ -0,0 +1,33 @@
+export default class Tasks {
+  static index = 0;
```

Comment:
- [ ] The issue of the index is coming from this static index it should actually be updated anytime you add a new task